SmartDeviceLink/public/SDLSecurityQueryErrorCode.h

```diff
@@ -0,0 +1,63 @@
+//
```

Comment:
This new file was placed in the "public" folder, but it's a project file and non-public, so this project no longer works in Cocoapods. Please move this class' files to the "private" folder and fix the project.